### PR TITLE
Roadmap: add Ongoing and emojis

### PR DESCRIPTION
**Pull request by psiinon**
OK, they should probably be in the template, but this works for now ;)

Screenshot 2022-09-29 at 15-40-19 OWASP ZAP – Roadmap

Signed-off-by: Simon Bennetts <psiinon@gmail.com>

### CODE REVIEW

**Review by thc202 — APPROVED** [review]

File: site/data/roadmap.yaml [hunk 1]
```diff
@@ -1,55 +1,55 @@
 - item: Networking overhaul
-  status: Finished
+  status: 🎉 Finished
   url: https://www.zaproxy.org/blog/2022-02-10-new-zap-networking-layer/
   year: 2022
 - item: Continue working on issues
-  status: In progress
+  status: 🚧 In progress
```
Comment (kingthorin) [inline site/data/roadmap.yaml:6]: Maybe 👷 instead or 🕑

This one kinda makes it seem like a road block.....

Comment (psiinon) [reply site/data/roadmap.yaml:6]: I think the clock is too similar to the calendar and didnt want a specific character for inclusiveness...

Comment (psiinon) [reply site/data/roadmap.yaml:6]: ⚒️⌨️🛣🪜?

Comment (psiinon) [reply site/data/roadmap.yaml:6]: 🎬⚡

Comment (kingthorin) [reply site/data/roadmap.yaml:6]: ⚒️ or ⚡ get my vote.

**Comment by psiinon** [comment]
Updated:

Screenshot 2022-09-29 at 16-15-21 OWASP ZAP – Roadmap

**Review by kingthorin — APPROVED** [review]

**Pull request merged**